linux_os/guide/services/xwindows/disabling_xwindows/xwindows_runlevel_setting/oval/shared.xml

```diff
@@ -35,7 +35,7 @@
   </unix:symlink_object>
   <unix:symlink_state comment="default.target is linked to multi-user.target" id="state_disable_xwindows_runlevel" version="1">
     <unix:filepath>/etc/systemd/system/default.target</unix:filepath>
-    <unix:canonical_path operation="pattern match">^/lib/systemd/system/multi-user.target$</unix:canonical_path>
+    <unix:canonical_path operation="pattern match">^/usr/lib/systemd/system/multi-user.target$</unix:canonical_path>
```

Comment:
Any idea when this change was made? Meaning should we create checks for RHEL6-7 using the old path, and RHEL8 for the new?

Reply:
In my tests, RHEL7.7 demonstrated the same behavior of RHEL8.X. Though the old location is still valid and the file exists, when using `systemctl set-default multi-user.target` it uses the new location with the prefix `/usr/`. I guess it needs to be extended to check for both locations if it is still a valid use case. I'll do some more testing around this to confirm all possible scenarios.

Reply:
I have extended the check to allow both locations since both ways work. I have tested in RHEL7.1 and RHEL7.7 the behavior and on a fresh installed system the target points to `/lib` but as soon as the default target is changed by `systemctl` it points to `/usr`

```
[0 root@ci-vm-10-0-137-213 ~]# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.1 (Maipo)
[0 root@ci-vm-10-0-137-213 ~]# systemctl get-default
multi-user.target
[0 root@ci-vm-10-0-137-213 ~]# ls -la /etc/systemd/system/default.target
lrwxrwxrwx. 1 root root 37 Jul 17  2014 /etc/systemd/system/default.target -> /lib/systemd/system/multi-user.target
[0 root@ci-vm-10-0-137-213 ~]# systemctl set-default multi-user.target
rm '/etc/systemd/system/default.target'
ln -s '/usr/lib/systemd/system/multi-user.target' '/etc/systemd/system/default.target'
[0 root@ci-vm-10-0-137-213 ~]# ls -la /etc/systemd/system/default.target
lrwxrwxrwx. 1 root root 41 Apr 23 11:02 /etc/systemd/system/default.target -> /usr/lib/systemd/system/multi-user.target
[0 root@ci-vm-10-0-137-213 ~]# systemctl get-default
multi-user.target






[root@ci-vm-10-0-136-215 ~]# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.7 (Maipo)
[root@ci-vm-10-0-136-215 ~]# systemctl get-default
multi-user.target
[root@ci-vm-10-0-136-215 ~]# ls -la /etc/systemd/system/default.target
lrwxrwxrwx. 1 root root 37 Aug  6  2019 /etc/systemd/system/default.target -> /lib/systemd/system/multi-user.target
[root@ci-vm-10-0-136-215 ~]# systemctl set-default multi-user.target
Removed symlink /etc/systemd/system/default.target.
Created symlink from /etc/systemd/system/default.target to /usr/lib/systemd/system/multi-user.target.
[root@ci-vm-10-0-136-215 ~]# ls -la /etc/systemd/system/default.target
lrwxrwxrwx. 1 root root 41 Apr 23 11:07 /etc/systemd/system/default.target -> /usr/lib/systemd/system/multi-user.target
[root@ci-vm-10-0-136-215 ~]# systemctl get-default
multi-user.target
```

Reply:
I confirm that `systemctl set-default` will prefer to user files from `/usr`. Fresh installed system has target pointing to `/lib`. So I updated the check to cover both cases.